app/util/AtomElementBuilders.scala

```diff
@@ -63,14 +64,49 @@ object AtomElementBuilders {
   }
 
   def buildHtml(atomType: AtomType, atomData: AtomData): Option[String] = atomType match {
-    case AtomType.Storyquestions => buildStoryQuestionsHtml(atomData)
+    case AtomType.Storyquestions => buildStoryQuestionsHtml(atomData.asInstanceOf[AtomData.Storyquestions].storyquestions)
```

Comment:
Not really an issue, but you can avoid using `asInstanceOf` by matching on the atomData, e.g. https://github.com/guardian/story-manager/blob/master/app/models/Atom.scala#L12